### ISSUE TITLE
[Tarea 3] Uso de random State

### ISSUE THREAD

**Issue by mmontecino** [IIC2613-Inteligencia-Artificial-2021-2/Syllabus#146]
Quería saber si es aceptado el uso de fijar un random_state dada la aleatoriedad en algunas funciones para poder así asegurarnos que no cambiaran los valores si se corre nuevamente.

**Comment by FeBalla** [comment]
Hola! Esto está en #131, usando ```np.random.seed(N)``` no deberían tener ese problema. De todos modos, además de usar eso, puedes usar también `random_state`, no hay ningún problema

**Comment by VicenteMerino** [comment]
@FeBalla creo que probando bien `np.random.seed` no funciona bien con todas las cosas de sklearn. Al menos con RF no me resultó reproducir los resultados y tuve que usar `random_state`.
